src/modules/differential_drive_control/DifferentialDriveKinematics/DifferentialDriveKinematics.cpp

```diff
@@ -0,0 +1,68 @@
+#include "DifferentialDriveKinematics.hpp"
+
+
+void DifferentialDriveKinematics::setInput(const matrix::Vector2f &input, bool inverse)
+{
+	_input = input;
+
+	if (inverse) {
+		_linear_vel_x = _input(0);
+		_yaw_rate = _input(1);
+		computeInverseKinematics();
+
+	} else {
+		_motor_vel_right = _input(0);
+		_motor_vel_left = _input(1);
+		computeForwardsKinematics();
+	}
+
+}
+
+void DifferentialDriveKinematics::computeInverseKinematics()
+{
+	_motor_vel_right = _linear_vel_x / _wheel_radius - _wheel_base / 2 * _yaw_rate / _wheel_radius;
```

Comment:
guard against /0

Reply:
Currently it guards against it in the setWheelBase and setWheelRadius, depending on if we keep this, I'll either add or not into computeInverseKinematics(...)